src/mmrelay/plugin_loader.py

```
@@ -870,485 +897,750 @@ def _check_auto_install_enabled(config):
+
+    Returns:
+        bool: `True` if the repository was updated to the commit, `False` otherwise.
+    """
+    try:
+        # If already at the requested commit, skip work
+        try:
+            current = _run_git(
+                ["git", "-C", repo_path, "rev-parse", "HEAD"], capture_output=True
+            ).stdout.strip()
+        except subprocess.CalledProcessError:
+            current = ""
```

Ruff (0.14.3)

Local variable `current` is assigned to but never used

Remove assignment to unused variable `current`

(F841)